build/Codecov.proj

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  
+  <Target Name="Codecov">
+    <PropertyGroup Condition="'$(UseCodecov)' == 'true'">
```

Comment:
You don't need the `UseCodecov` conditions here. If you want to have a condition, put it on the target.

Reply:
Makes sense.